test_lodcm_destination failing on master

See https://travis-ci.org/pcdshub/pcdsdevices/jobs/491757874

```
______________________________________________________________________________________________ test_lodcm_destination _______________________________________________________________________________________________

fake_lodcm = FakeLODCM(prefix='FAKE:LOM', name='fake_lom', settle_time=0.0, timeout=None, read_attrs=['state', 'readback'], configuration_attrs=[])

    def test_lodcm_destination(fake_lodcm):
        logger.debug('test_lodcm_destination')
        lodcm = fake_lodcm
        dest = lodcm.destination
        assert isinstance(dest, list)
        for d in dest:
            assert isinstance(d, str)

        lodcm.state.put('OUT')
        assert len(lodcm.destination) == 1
        lodcm.state.put('C')
        assert len(lodcm.destination) == 2
        # Block the mono line
>       lodcm.yag.state.put('IN')

tests/test_lodcm.py:49:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tests/conftest.py:43: in put
    super().put(value, *args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = HotfixFakeEpicsSignal(name='fake_lom_yag_state', parent='fake_lom_yag', value=1, timestamp=1551917464.099958), value = 'IN', args = (), kwargs = {}
err = "IN not in enum strs ['Unknown', 'OUT', 'YAG', 'SLIT1', 'SLIT2', 'SLIT3']"

    def put(self, value, *args, **kwargs):
        """
        Implement putting as enum strings and put functions
        """
        if self.enum_strs is not None:
            if value in self.enum_strs:
                value = self.enum_strs.index(value)
            elif isinstance(value, str):
                err = '{} not in enum strs {}'.format(value, self.enum_strs)
>               raise ValueError(err)
E               ValueError: IN not in enum strs ['Unknown', 'OUT', 'YAG', 'SLIT1', 'SLIT2', 'SLIT3']

../ophyd/ophyd/sim.py:1119: ValueError
```

--- a/tests/test_lodcm.py
+++ b/tests/test_lodcm.py
@@ -41,16 +41,16 @@ def test_lodcm_destination(fake_lodcm):
     for d in dest:
         assert isinstance(d, str)
 
-    lodcm.state.put('OUT')
+    lodcm.move('OUT')
     assert len(lodcm.destination) == 1
-    lodcm.state.put('C')
+    lodcm.move('C')
     assert len(lodcm.destination) == 2
     # Block the mono line
-    lodcm.yag.state.put('IN')
+    lodcm.yag.move('IN')
     assert len(lodcm.destination) == 1
-    lodcm.state.put('Si')
+    lodcm.move('Si')
     assert len(lodcm.destination) == 0
-    lodcm.yag.state.put('OUT')
+    lodcm.yag.move('OUT')
     assert len(lodcm.destination) == 1
 
     # Unknown state

